nemo/collections/asr/metrics/wer.py

```diff
@@ -95,6 +95,7 @@ def validation_epoch_end(self, outputs):
         use_cer: Whether to use Character Error Rate instead of Word Error Rate.
         ctc_decode: Whether to use CTC decoding or not. Currently, must be set.
         log_prediction: Whether to log a single decoded sample per call.
+        fold_consecutive: Whether repeated consecutive characters should be folded into one when decoding.
```

Comment:
I don't think there's a need to modify / create new `word_error_rate` because there's no scenario that involves it with k2 models. I'll come back to it in future upgrades if necessary.